core/cmd/client_test.go

```diff
@@ -329,3 +331,59 @@ func TestNewUserCache(t *testing.T) {
 	assert.DirExists(t, c.RootDir())
 
 }
+
+func TestSetupSolanaRelayer(t *testing.T) {
+	lggr := logger.TestLogger(t).Named("solana-test")
+	db := pgtest.NewSqlxDB(t)
+	cfg := configtest.NewGeneralConfig(t, nil)
+	reg := plugins.NewLoopRegistry()
+	ks := &mocks.Solana{}
```

Comment:
Please use the constructors for mocks:
```suggestion
	ks := mocks.NewSolana(t)
```
It prevents panics by wiring failures to the `t`, and also sets up cleanup for you.